src/components/textfield/textfield.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import styled from 'styled-components';
+import { css } from 'styled-components';
+
+const TextXS = css`
+  font-size: 12px;
+  line-height: 150%;
+`;
+
+const TextR = css`
+  font-size: 16px;
+  line-height: 150%;
+`;
+
+const StyledLabel = styled.span`
+  ${TextXS}
+  display: block;
+  margin-bottom: 4px;
+
+  letter-spacing: -0.01em;
+  color: #202840;
+`;
+
+const StyledErrorMessage = styled.span`
+  display: block;
+  margin-top: 4px;
+
+  ${TextXS}
+  letter-spacing: -0.01em;
+  font-weight: 700;
+  color: #ff0d35;
+`;
```

Comment:
For the sake of consistency I suggest to keep the `${TextXS}` at the very top of these two components. If they both start with the following three lines then it's immediately visible that they basically are very similar.
```css
  ${TextXS}
  display: block;
  margin-...
```

According to Figma the `letter-spacing` is also part of `TextXS` so please move that into `TextXS` to avoid duplication.